system/settings/settings.xml

```diff
@@ -2941,6 +2941,20 @@
           <default>false</default>
           <control type="toggle" />
         </setting>
+        <setting id="addons.prefupdatemode" type="integer" label="36422" help="36423">
```

Comment:
Why not just called `addons.updatemode`?

Or if is needs to be qualified: `addons.trustedupdatemode`

Reply:
I think you also need to set the parent attribute here to `addons.unknownsources`

Reply:
ok, done